spec/tictactoe_spec.rb

```diff
@@ -1,48 +1,36 @@
 require "spec_helper"
 require "tictactoe"
+require "fake_display"
 
 RSpec.describe TicTacToe do
-  class TicTacToe::FakeDisplay
-    attr_reader :lines
-
-    def initialize
-      @lines = []
-    end
-
-    def output(message)
-      @lines << message
-      nil
-    end
-
-    def input
-      :not_used
-    end
-  end
-
   class FakePresenter
     def display_board(board)
       (1..9).map { |i| board.get(i) || "-" }.join("")
     end
   end
 
   class FakePlayer
-    def selection
-      1
+    def initialize(display)
+      @display = display
+    end
```

Comment:
Does the `FakePlayer` need a display? From my own experience, I don't recall building test doubles that accept anything more than some initial state in the initializer. `display` seems to be more of a collaborator than state.

Reply:
It probably doesn't.  Will adjust.